npm-shrinkwrap.json

```diff
@@ -498,26 +515,27 @@
         },
         "caseless": {
           "version": "0.8.0",
-          "from": "caseless@~0.8.0",
+          "from": "https://registry.npmjs.org/caseless/-/caseless-0.8.0.tgz",
           "resolved": "https://registry.npmjs.org/caseless/-/caseless-0.8.0.tgz"
         },
         "forever-agent": {
           "version": "0.5.2",
-          "from": "forever-agent@~0.5.0"
+          "from": "forever-agent@0.5.2",
+          "resolved": "https://registry.npmjs.org/forever-agent/-/forever-agent-0.5.2.tgz"
         },
         "form-data": {
           "version": "0.2.0",
-          "from": "form-data@~0.2.0",
+          "from": "https://registry.npmjs.org/form-data/-/form-data-0.2.0.tgz",
           "resolved": "https://registry.npmjs.org/form-data/-/form-data-0.2.0.tgz",
           "dependencies": {
             "mime-types": {
               "version": "2.0.7",
-              "from": "mime-types@~2.0.3",
+              "from": "https://registry.npmjs.org/mime-types/-/mime-types-2.0.7.tgz",
```

Comment:
After a second look at this, `version` and `resolved` haven't changed, so it seems to be ok. NTC.